fre/make/gfdlfremake/checkout.py

```diff
@@ -115,6 +115,7 @@ def run (self):
         Param:
             - self The checkout script object
         """
+        os.chmod(self.src+"/"+self.fname, 0o744)
```

Comment:
I don't think this would be needed here. The checkout script being made executable is now handled in the `createCheckout` tool: https://github.com/NOAA-GFDL/fre-cli/blob/c82680493b2a650b36011f0614c22b62117b31c6/fre/make/createCheckout.py#L78

Reply:
Users want the checkout script to be made executable for the bare-metal build before actually being able to run it. I made that change in the `createChecout.py` tool and forgot to change it for `run_fremake` as well. I would put this `chmod` line here: https://github.com/NOAA-GFDL/fre-cli/blob/34eb37e2130ddf4703f343f0ccc9bcc38d2102b8/fre/make/runFremake.py#L92

Right after the checkout script gets created.

Reply:
This should be fixed now with commit ee4b5b5. Will run-fremake not always execute everything? I think thats what it does currently.